src/components/Sentence/index.js

```diff
@@ -16,52 +16,64 @@ const Sentence = React.createClass({
     'foamy',
     'primitive',
     'blathery',
-    'crumpled'
+    'crumpled',
+    'tiny'
   ],
-  nouns: [
+  nounsSubject: [
     'handkerchief',
-    'blognoggin',
+    'elephant tusk',
     'umbrella',
     'big foot',
-    'blisterbubble'
+  ],
+  nounsObject: [
+    'fancily decorated toothpick',
+    'bunch of blue balloons',
+    'washing machine lint filter with towel fluff in it',
+    'banana with a brown spot'
+
+
   ],
   verbs: [
     'skitter',
     'froth',
     'zoom forward',
     'stew',
-    'operate',
-    'stomp'
-  ],
-  adverbs: [
-    'speedily',
-    'clammily',
-    'frighteningly',
-    'knowingly',
-    'blisteringly'
+    'waggle its little finger',
+    'stomp',
+    'chew a purple gumball',
+    'fray'
   ],
+  // adverbs: [
+  //   'speedily',
+  //   'clammily',
+  //   'frighteningly',
+  //   'knowingly',
+  //   'blisteringly',
+  //   'alarmingly'
+  // ],
   pronouns: [
     'am I',
     'art thou',
     'is thee',
-    'art we',
-    'ist he',
-    'ist she',
-    'ist it'
+    'are we',
+    'is he',
+    'is she',
+    'is it'
   ],
   possessives: [
-    'mine',
-    'thy',
-    'thine',
+    'my',
+    'your',
+    'their',
     'his',
-    'her'
+    'her',
+    'our'
   ],
-  getWord: function(posp){
-    const index = Math.floor(Math.random() * posp.length);
-    return posp[index];
+  getWord: function(poSp){
```

Comment:
um, I was going to ask you what `poSp` was. I think `poSpArr` is even more cryptic! Let's try for a very clear name that doesn't require a special decoder ring to decipher :)